### PR TITLE
Use new ftw theming vars

### PR DESCRIPTION
**Pull request by elioschmutz**
``$border-radius`` is no longer available in ftw.theming.

update it to ``$border-radius-primary``

@maethu @bierik pls take a look

### CODE REVIEW

**Comment by elioschmutz** [comment]
@maethu @bierik could someone take a look at this branch and merge it?

File: ftw/solr/browser/resources/ftwtheming.scss [hunk 1]
```diff
@@ -179,7 +179,7 @@ $dropdown-link-color-hover: $link-hover-color !default;
 
         @include boxshadow(0.5em 1em 2em -1em $dropdown-border-color);
 
-        @include borderradius($border-radius);
+        @include borderradius($border-radius-primary);
```
Comment (bierik) [inline ftw/solr/browser/resources/ftwtheming.scss]: `borderradius` mixin is not necessary. According to http://caniuse.com/#search=border-radius we need no vender prefixes for `border-radius`.

Comment (maethu) [inline ftw/solr/browser/resources/ftwtheming.scss]: So, only `border-radius: $border-radius-primary`is needed?

**Comment by elioschmutz** [comment]
@bierik :+1: updated

**Pull request merged**